hibernate-core/src/test/java/org/hibernate/test/joinedsubclassbatch/Person.hbm.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
```

Comment:
Better use annotations and embed the entity in the test class itself.

Reply:
It was a copy paste from an existing test related to batching where i just added my annotations